themes/classic/modules/ps_shoppingcart/modal.tpl

```diff
@@ -32,70 +32,64 @@
+          {/if}
+          <p><span class="label">{l s='Subtotal:' d='Shop.Theme.Checkout'}</span>&nbsp;<span class="subtotal value">{$cart.subtotals.products.value}</span></p>
+          {if $cart.subtotals.shipping.value}
+            <p><span>{l s='Shipping:' d='Shop.Theme.Checkout'}</span>&nbsp;<span class="shipping value">{$cart.subtotals.shipping.value} {hook h='displayCheckoutSubtotalDetails' subtotal=$cart.subtotals.shipping}</span></p>
+          {/if}
+          {if !$configuration.display_prices_tax_incl && $configuration.taxes_enabled}
+            <p><span>{$cart.totals.total.label}{if $configuration.display_taxes_label}&nbsp;{$cart.labels.tax_short}{/if}</span>&nbsp;<span>{$cart.totals.total.value}</span></p>
+            <p class="product-total"><span class="label">{$cart.totals.total_including_tax.label}</span>&nbsp;<span class="value">{$cart.totals.total_including_tax.value}</span></p>
+          {else}
+            <p class="product-total"><span class="label">{$cart.totals.total.label}&nbsp;{if $configuration.taxes_enabled && $configuration.display_taxes_label}{$cart.labels.tax_short}{/if}</span>&nbsp;<span class="value">{$cart.totals.total.value}</span></p>
+          {/if}
 
-              {if $cart.subtotals.tax}
-                <p class="product-tax">{l s='%label%:' sprintf=['%label%' => $cart.subtotals.tax.label] d='Shop.Theme.Global'}&nbsp;<span class="value">{$cart.subtotals.tax.value}</span></p>
-              {/if}
-              {hook h='displayCartModalContent' product=$product}
-              <div class="cart-content-btn">
-                <button type="button" class="btn btn-secondary" data-dismiss="modal">{l s='Continue shopping' d='Shop.Theme.Actions'}</button>
-                <a href="{$cart_url}" class="btn btn-primary"><i class="material-icons rtl-no-flip">&#xE876;</i>{l s='Proceed to checkout' d='Shop.Theme.Actions'}</a>
-              </div>
-            </div>
-          </div>
+          {if $cart.subtotals.tax}
+            <p class="product-tax">{l s='%label%:' sprintf=['%label%' => $cart.subtotals.tax.label] d='Shop.Theme.Global'}&nbsp;<span class="value">{$cart.subtotals.tax.value}</span></p>
+          {/if}
+          {hook h='displayCartModalContent' product=$product}
         </div>
-      </div>
-      {hook h='displayCartModalFooter' product=$product}
+        <div class="line-break"></div>
+        <div class="cart-content-btn">
+          <button type="button" class="btn btn-secondary" data-dismiss="modal">{l s='Continue shopping' d='Shop.Theme.Actions'}</button>
+        </div>
+        <div class="cart-content-btn checkout-action">
+          <a href="{$cart_url}" class="btn btn-primary"><i class="material-icons rtl-no-flip">&#xE876;</i>{l s='Proceed to checkout' d='Shop.Theme.Actions'}</a>
+        </div>
+        <div class="line-break"></div>
+        {hook h='displayCartModalFooter' product=$product}
```

Comment:
```suggestion
        <div class="cart-content-btn">
          <button type="button" class="btn btn-secondary" data-dismiss="modal">{l s='Continue shopping' d='Shop.Theme.Actions'}</button>
        </div>
        <div class="cart-content-btn checkout-action">
          <a href="{$cart_url}" class="btn btn-primary"><i class="material-icons rtl-no-flip">&#xE876;</i>{l s='Proceed to checkout' d='Shop.Theme.Actions'}</a>
        </div>
        <div class="card-content-footer">
          {hook h='displayCartModalFooter' product=$product}
        </div>
```

I'm pretty sure you'll be able to have the same result with a markup like this, I don't really understand the need of some line-break element